### DISCUSSION TITLE
how to set javaopts in quarkus based keycloak

### DISCUSSION THREAD

**Discussion by kkcmadhu** [keycloak/keycloak#13070] [Archive]
hi both keycloak and quarkus documents do not mention much on how to pass java opts like xmx xms grabasecollector gc log and other parameters.. Can some one let me know  how this should be done?

**Comment by dasniko** [comment]
You can set the `JAVA_OPTS_APPEND` environment variable. It will, as the name says, append the specified opts to the default JAVA_OPTS variable.
